paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -832,7 +832,11 @@ class PaparazziPluginTest {
       "app.cash.paparazzi.plugin.test.module1",
       "app.cash.paparazzi.plugin.test.module2"
     )
-    assertThat(config.projectResourceDirs).containsExactly("build/generated/res/extra", "src/main/res", "src/debug/res", "build/generated/res/resValues/debug")
```

Comment:
is this removal correct?

Reply:
oops, i forgot to rebase on latest, which contains the AGP/tools bump necessary for the fix of this very issue: https://issuetracker.google.com/issues/331946533